src/main/java/se/sundsvall/supportmanagement/service/MetadataService.java

```diff
@@ -85,6 +87,7 @@ public MetadataService(final CategoryRepository categoryRepository,
 		this.statusRepository = statusRepository;
 		this.validationRepository = validationRepository;
 		this.contactReasonRepository = contactReasonRepository;
+		this.pathMatcher = new AntPathMatcher();
```

Comment:
Should we use case insensitive matching?
If yes: `this.pathMatcher.setCaseSensitive(false)`